src/applications/personalization/view-dependents/tests/components/ViewDependentsHeader.unit.spec.jsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import { expect } from 'chai';
 import { shallow } from 'enzyme';
-import ViewDependentsHeader from '../../components/ViewDependentsHeader/ViewDependentsHeader';
+import { ViewDependentsHeader } from '../../components/ViewDependentsHeader/ViewDependentsHeader';
 
 describe('<ViewDependentsHeader />', () => {
   it('Should Render', () => {
-    const wrapper = shallow(<ViewDependentsHeader />);
+    const url = path => path;
+    const wrapper = shallow(<ViewDependentsHeader eBenefitsUrl={url} />);
```

Comment:
Since `ViewDependentsHeader` isn't a container anymore, I think we can test using the default export